WPMedia/fusion-news-theme-blocks
# Themes 1440

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Updated Twitter icon on hover

## Jira Ticket

- [THEMES-1440](https://arcpublishing.atlassian.net/browse/THEMES-1440)

## Acceptance Criteria
Update Twitter icon on hover

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-1440`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmwdia/share-bar-block`
3. check for twitter icon

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1440]: https://arcpublishing.atlassian.net/browse/THEMES-1440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
